src/app/components/header/header.component.html

```diff
@@ -6,6 +6,8 @@
     <exui-hmcts-global-header
     [headerTitle]="title"
     [navigation]="userNav"
+    [showFindCase]="showFindCase"
```

Comment:
@johnbenjamin-hmcts - the `showFindCase` flag drops out of use following @ronaldmansveld's changes from PR #1396.